Gemfile.lock

```diff
@@ -299,6 +299,3 @@ DEPENDENCIES
   unicorn
   vcr
   whenever
-
-BUNDLED WITH
```

Comment:
After v1.10, bundler adds it automatically. I was using an older version which removes it on ```bundle install```